packages/builder/src/actions.ts

```diff
@@ -82,11 +58,15 @@ export function registerAction(action: CannonAction) {
   }
 
   ActionKinds[label] = action;
-  (ChainDefinitionSchema.optionalProperties as any)[label] = { values: action.validate };
-}
 
-export function getChainDefinitionValidator() {
-  return ajv.compile(ChainDefinitionSchema);
+  (
+    chainDefinitionSchema.pick({
+      description: true,
+      keywords: true,
+      setting: true,
+      import: true,
+    }) as any
```

Comment:
what is the purpose of using `.pick` and then `as any` afterwards here?

Reply:
Was trying to match the way it was being done before, also, as any was the only way I could index the object returned by `pick` with the `label` string